app/src/main/java/com/traveloka/hotel/ui/navigation/HotelNavigation.kt

```diff
@@ -26,6 +28,13 @@ fun HotelNavigation() {
         composable(HotelScreens.ListHotelScreen.name) {
             ListHotelScreen(navController)
         }
+        composable(HotelScreens.DetailsScreen.name){
+            val hotel=navController.previousBackStackEntry?.savedStateHandle?.get<Hotel>("hotel")
+
+            hotel?.let {
+                DetailHotelScreen(navController, hotel)
+            }
+        }
```

Comment:
```suggestion
        composable(HotelScreens.DetailsScreen.name+"/{hotelId}", arguments = listOf(navArgument("hotelId") { type = NavType.StringType })){ backStackEntry ->
            DetailHotelScreen(navController, backStackEntry.arguments?.getString("hotelId"))
        }
```


ini coba ikutin kek di [docs ](https://developer.android.com/jetpack/compose/navigation#nav-with-args) aja, klo case ini passing idnya aja tar di detail screennya fetch API hotel by id buat get detail hotelnya jgn parceable

Reply:
Oke, thanks